src/jit/codegenxarch.cpp

```diff
@@ -6184,6 +6201,14 @@ void CodeGen::genCallInstruction(GenTreePtr node)
 
 #endif // defined(_TARGET_X86_)
 
+    if (call->IsTailCallViaHelper())
+    {
+        if (compiler->getNeedsGSSecurityCookie())
+        {
+            genEmitGSCookieCheck(true);
+        }
+    }
```

Comment:
I know.  But I thought it is better not to execute code which is not required in the first place.  Besides, #ifdef is very nearby :)